src/Sylius/Bundle/CoreBundle/Form/Extension/ShippingMethodTypeExtension.php

```diff
@@ -28,36 +26,6 @@
 final class ShippingMethodTypeExtension extends AbstractTypeExtension
 {
     /**
-     * @var ServiceRegistryInterface
-     */
-    private $checkerRegistry;
-
-    /**
-     * @var ServiceRegistryInterface
-     */
-    private $calculatorRegistry;
-
-    /**
-     * @var FormTypeRegistryInterface
-     */
-    private $formTypeRegistry;
-
-    /**
-     * @param ServiceRegistryInterface $checkerRegistry
-     * @param ServiceRegistryInterface $calculatorRegistry
-     * @param FormTypeRegistryInterface $formTypeRegistry
-     */
-    public function __construct(
-        ServiceRegistryInterface $checkerRegistry,
-        ServiceRegistryInterface $calculatorRegistry,
-        FormTypeRegistryInterface $formTypeRegistry
-    ) {
```

Comment:
Let's make sure we delete those from the service definition.

Reply:
I guess we should mention about these changes in upgrade file. Even if it was not used it is a BC at the end